ray-operator/pkg/utils/string_conversion.go

```diff
@@ -0,0 +1,17 @@
+// Util functions for type conversion between byte array and string.
+
+package utils
+
+import (
+	"unsafe"
+)
+
+// Convert a byte array to string w/o copy.
```

Comment:
Can you add some comments to explain that this is not safe if the underlying byte array is modified?

Reply:
Sure, comments added.